src/object/clone.spec.ts

```diff
@@ -238,4 +238,28 @@ describe('clone', () => {
     expect(clonedError.message).toBe(error.message);
     expect(clonedError.name).toBe(error.name);
   });
+
+  it('should clone class instance', () => {
```

Comment:
I think this test code is added in `clone`, not `cloneDeep`. Let me add a test case for `cloneDeep`, too.

Reply:
There already is a `should clone instance` test in `cloneDeep.spec.ts`, so that's why I only added a single test covering this case.